docs/external-node.md

```diff
@@ -277,7 +281,7 @@ please refer to the [getting-started guide](getting-started.md#open-vswitch).
 3. Bootstrap `antrea-agent` using one of these 2 methods:
 
    1. Bootstrap `antrea-agent` using the [installation script](../hack/externalnode/install-vm.sh)
-      as shown below (Ubuntu 18.04 and 20.04 only).
+      as shown below (Ubuntu 18.04 and 20.04, Rhel 8.4).
```

Comment:
I feel better to use full name for Redhat Enterprise Linux

Reply:
Done.